simapp/go.mod

```diff
@@ -204,6 +204,7 @@ replace (
 
 // Below are the long-lived replace of the SimApp
 replace (
+	cosmossdk.io/x/tx => ../x/tx
```

Comment:
tiny tiny nit: we have two replaces blocks now, one for temporary replace like this one, and one for long lived replaces.

Reply:
BTW, we should just tag a new version of API, instead of using a replace.

Reply:
API? I think you mean `x/tx`?

Reply:
Oh yeah I meant `x/tx`. I wasn't sure if the x/tx changes here were necessary (https://github.com/cosmos/cosmos-sdk/pull/14994#discussion_r1104469064), I was thinking no, that's why I proposed to tag x/tx